### PR TITLE
quiz branch merge

### PR DESCRIPTION
**Pull request by pbhowelljr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	<script src="quiz.js" type="text/javascript" charset="utf-8" async defer></script>
+</head>
+<body>
+	<input id="heightInput" type="text" placeholder="Height">
+	<input id="characterInput" type="text" placeholder="Character">
+	<button id="submitButton"></button>
+	<button id="clearButton"></button>
+	<div id="mainContent"></div>
+</body>
+</html>
```
Comment (NewEvolution) [inline index.html]: No newline at end of flie.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	height: 60px;
+	width: 75px;
+	cursor: pointer;
+}
+
+#clearButton {
+	height: 25px;
+	width: 75px;
+	cursor: pointer;
+}
```
Comment (NewEvolution) [inline main.css]: No newline at end of file.

File: quiz.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+var clear = function () {
+	mainContent.innerHTML = "";
+	heightInput.value = "";
+	characterInput.value = "";
+};
+
+
+//PRINTS FORMATTED ARRAY TO DOM AND CONSOLE
+var arrayPrinter = function(array, elem) {
+	var arrayString = array.join(" ").toString();
```
Comment (NewEvolution) [inline quiz.js]: If you look at the example output, there are no spaces between the characters making up the tree.

File: quiz.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+	elem.innerHTML += ('<div>'+arrayString+'</div>');
+	console.log(arrayString);
+};
+
+
+//RUNS RUNNER IF ENTER KEY WAS PRESSED INSIDE A TEXT INPUT
+var enterChecker = function(key) {
+	if ((key.which===13) || (key.keyCode==13)) {
```
Comment (NewEvolution) [inline quiz.js]: Should use the === on the second comparison as well.

File: quiz.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+var treeBuilder = function(specs, array) {
+	var character = specs.character;
+	var height = specs.height;
+	var middle=(height-1);
+
+	for (var i = 0; i < (((height-1)*2)+1); i++) {
+		array.push(' '.repeat(character.length));
+	}
+
+	array[(middle)] = character;
```
Comment (NewEvolution) [inline quiz.js:64]: The parentheses are extraneous.

**Comment by pbhowelljr** [comment]
All comments addressed. thank you. (I am sorry I didn't leave a space at the bottom of the html and css, you will not have to tell me again.)

**Comment by NewEvolution** [comment]
👍  Awesome.  Really impressed with the nested function calls etc.  Shows a great grasp of tossing functions around, you'll probably have no issues once we get to callback functions etc.

**Pull request merged**